treeherder/client/thclient/client.py

```diff
@@ -674,16 +674,16 @@ def get_machine_platforms(self):
         """
         return self._get_json(self.MACHINE_PLATFORM_ENDPOINT)
 
-    def get_resultsets(self, project, **params):
+    def get_pushes(self, project, **params):
```

Comment:
we should add a deprecated reference to this function called ``get_resultsets`` that calls this for backward compatibility.